vpr/src/draw/draw_basic.cpp

```diff
@@ -0,0 +1,1227 @@
+/*draw_basic.cpp contains all functions that draw things on the screen.*/
```

Comment:
Can you be more specific?  I assume draw things means draw in the main graphics/visualization area?

Reply:
Also, it seems this comment isn't correct, as there are also a draw_rr and draw_mux files.